src/Main/Main.js

```diff
@@ -1,17 +1,24 @@
 import './Main.css';
 import Spells from '../Spells/Spells.js';
+import { NavLink } from 'react-router-dom';
```

Comment:
navlink is crazy useful and this a phenomenal implementation of it, good thinking